psh/dd/dd.c

```diff
@@ -429,6 +436,10 @@ static int psh_dd(int argc, char **argv)
 		(size_t)(outtotal / blocksz),
 		(int)(outtotal % blocksz) != 0);
 
+	elapsed_time = (end_time.tv_sec - start_time.tv_sec) + (end_time.tv_usec - start_time.tv_usec) / 1000000.0f;
+	fprintf(stderr, "%zu byte%s copied, %.3f s, %.1f kB/s\n", (size_t)intotal, ((intotal != 1) ? "s" : ""),
+		elapsed_time, (float)intotal / elapsed_time / 1024.0f);
+
 	return EXIT_FAILURE;
```

Comment:
Out of this PR but probably should be:
```suggestion
	return EXIT_SUCCESS;
```

Reply:
Fixed, thanks.